docker/Dockerfile

```diff
@@ -1,21 +1,18 @@
-FROM node:18-alpine as build
+FROM node:20-alpine as build
 
 RUN apk --no-cache add g++ gcc libgcc libstdc++ linux-headers make python3 git
-RUN npm install --quiet node-gyp -g
 WORKDIR /app
 
 COPY package.json ./
-COPY yarn.lock ./
-COPY .yarn ./app/
-COPY .yarnrc ./app/
-RUN yarn
+COPY package-lock.json ./
+RUN npm ci
```

Comment:
@GerwinBosch Why did you change this? The repo has no `package-lock.json`, and now seems to use Yarn instead of NPM, so this fails.

Reply:
Whoops, I guess I was an oversight from me. Now that the LD-Wizard base package uses NPM instead of yarn (https://github.com/pldn/LDWizard/issues/158), the build script, which copies the dockerfile, was also changed to use NPM.

I don't have a solution at hand other than manually overriding or it or complicating the build script to detect the used package manager

Reply:
I've reverted the docker-file back to use Yarn, it now fails at the same point as the last commit (https://github.com/netwerk-digitaal-erfgoed/LDWizard-ErfgoedWizard/actions/runs/7100619776/attempts/1)

Reply:
Why revert back to Yarn? Let’s just use NPM: #58.